dwarf-fortress.sh

```diff
@@ -27,9 +27,11 @@ while getopts ":hsdR" opt; do
     case $opt in
         h)
```

Comment:
My suggestion; extend the `getopts` and add a 't' option as well, only set `TWBT="true"` on it.

Since the check for the feature explicitly looks for the string "true", an unset variable won't activate TWBT.

Reply:
Updated, PTAL. Not entirely sure if `getopts` string is correct though